pyaerocom/_lowlevel_helpers.py

```diff
@@ -410,7 +410,7 @@ def _class_name(self):
         return _class_name(self)
 
     def keys(self):
-        return list(self.__dict__.keys()) + self.ADD_GLOB
+        return list(self.__dict__) + self.ADD_GLOB
```

Comment:
isn't `self.__dict__` a dictionary? [The documentation](https://docs.python.org/3/library/stdtypes.html#object.__dict__) is suggesting that. Just to be sure